python/ray/serve/utils.py

```diff
@@ -107,6 +107,74 @@ def parse_request_item(request_item):
     return request_item.args, request_item.kwargs
 
 
+class LoggingContext:
+    r"""
```

Comment:
we don't need to r string here, most of the Ray codebase use simple multi-line string for comments.
```suggestion
    """
```
```suggestion
    r"""
```